lib/gateway/Shard.js

```diff
@@ -1014,10 +1014,17 @@ class Shard extends EventEmitter {
                         --this.getAllUsersCount[guild.id];
                     }
                 }
-                packet.d.members.forEach((member) => {
+
+                /**
+                 * Fired when discord sends members on startup
```

Comment:
Awkward wording. Maybe something like `Fired when Discord sends member chunks`

This also isn't limited to just startup